File: frontend/src/app/dto/learning-schedule/learning-schedule.insert.res.dto.ts
import { LearningScheduleInsertDataResDto } from "./learning-schedule.insert.data.res.dto";

export class LearningScheduleInsertResDto{
    private _data! : LearningScheduleInsertDataResDto
	private _message! : string

    public get data(): LearningScheduleInsertDataResDto {
        return this._data;
    }

    public set data(data: LearningScheduleInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}

File: src/main/java/com/lawencon/elearn/dto/learningschedule/LearningScheduleInsertResDto.java
package com.lawencon.elearn.dto.learningschedule;

public class LearningScheduleInsertResDto {
	private LearningScheduleInsertDataResDto data;
	private String message;
	public LearningScheduleInsertDataResDto getData() {
		return data;
	}
	public void setData(LearningScheduleInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}
